File: trunk/quests/templeveeshan/#Lord_Kreizenn.pl
# link lord kreizenn and guards
#

sub EVENT_AGGRO {
  my $guard1 = $entity_list->GetMobByNpcTypeID(124093); #kedrak
  my $guard2 = $entity_list->GetMobByNpcTypeID(124094); #carx`vean
  if ($guard1) {
    my $call_guard1 = $guard1->CastToNPC();
    $call_guard1->AddToHateList($client, 1);
  }
  if ($guard2) {
    my $call_guard2 = $guard2->CastToNPC();
    $call_guard2->AddToHateList($client, 1);
  }
}

# EOF zone: templeveeshan ID: 124074 NPC: #Lord_Kreizenn

File: trunk/quests/templeveeshan/#Lord_Kreizenn.lua
-- #Lord_Kreizenn NPCID: 124074
function event_combat(e)
	if (e.joined) then
		-- grab the entity list
		local entity_list = eq.get_entity_list();
		-- also aggro Kedrak (124093) and Carx`Vean (124094) if they are up
		local npc_table = {124093,124094};
		for k,v in pairs(npc_table) do
			local npc = entity_list:GetMobByNpcTypeID(v);
			if (npc.valid) then
				npc:AddToHateList(e.other,1);
			end
		end
	end
end